src/components/Canvas.tsx

```diff
@@ -22,6 +23,8 @@ const Canvas: React.VFC<Props> = (props) => {
   const [lineColor, setLineColor] = useState("#000000");
   const [tool, setTool] = useState<ToolType>("pen");
   const isDrawing = useRef<boolean>(false);
+  const [history, setHistory] = useState<LineType[][]>([]);
```

Comment:
historyの初期値の変更についてです。

```suggestion
  const [history, setHistory] = useState<LineType[][]>([[]]);
```
個人的には、historyの初期値は`[]`ではなく`[[]]`のほうがいいと思ってます。
（ちなみに`[[]]`の中身の`[]`は初期状態の真っ白のキャンバスを表しています。）

理由は、
- redo, undoによって、これまでの全ての状態を再現できるのが望ましい
- そのため、historyは初期状態から始まるべきである